internal/ui/tabs.go

```diff
@@ -31,6 +31,8 @@ func Create(app fyne.App, window fyne.Window) *container.AppTabs {
 			tabs.SelectIndex(0)
 		}
 
+		send.client.CustomCode = true
```

Comment:
Sure thing, would be consistent & intuitive too. Though I might need more time to explore Fyne though 😅 .

Reply:
Cool. I think most of what needs doing should be similar to code that is already there. However, you are of course more than welcome to ask if you need any help :)

Reply:
Hi @Jacalz, I've made an update. Now when file is being dropped, a prompt is shown. But I can't set the `CustomCode` properly back to `false` after it's being set to `true`:
```go
confirm := dialog.NewConfirm("Create using custom code", "Create using custom code?", func(custom bool) {
	send.client.CustomCode = custom
	send.newTransfer(uris)
        send.client.CustomCode = false // -> won't work because the newTransfer is being done concurrently (I guess)
         }, window)
confirm.Show()
```

This will cause a side effect: the `Use a custom code` tick will not be ticked but the state of the `send.client.CustomCode` is still set to false.

So in order to mitigate this problem, I'm updating the state of the whole window, by newly creating it every time the `Add content to send` button is tapped:
```go
contentToSend := &widget.Button{Text: "Add content to send", Icon: theme.ContentAddIcon(), OnTapped: func() {
		codeChoice := &widget.Check{Text: "Use a custom code", OnChanged: s.onCustomCode, Checked: s.client.CustomCode}
		choiceContent := container.NewGridWithColumns(1, fileChoice, directoryChoice, textChoice, codeChoice)
		s.contentPicker = dialog.NewCustom("Pick a content type", "Cancel", choiceContent, window)
		s.contentPicker.Show()
	}}
```

 I'm not sure about the performance or how proper the behavior is, so kindly give feedbacks on this. 

Many thanks!

Reply:
Thanks. The solution does look very good visually and from a usability standpoint. I will have a look at if the state problem you are talking about can be improved.